src/components/dev-dot-content/mdx-components/mdx-hcp-callout/mdx-callout.module.css

```
@@ -0,0 +1,66 @@
+.root {
+	width: 100%;
+	background: var(--black);
+	color: var(--white);
+	border-radius: 5px;
+	display: flex;
+	position: relative;
+	overflow: hidden;
+	box-shadow: var(--token-elevation-mid-box-shadow);
+	margin: 32px 0;
+
+	&:hover {
+		box-shadow: var(--token-elevation-high-box-shadow);
+
+		& .ctaIcon {
+			transform: translateX(2px);
+		}
+	}
+
+	& .textContainer {
+		padding: 16px 22px;
+	}
+
+	& .heading {
+		/* !important is necessary here to stop leaks from enclosing
+     @hashicorp/react-content component styles */
+		margin: 0 !important;
+	}
+
+	& .subHeading {
+		/* !important is necessary here to stop leaks from enclosing
+     @hashicorp/react-content component styles */
+		margin: 6px 0 16px !important;
+	}
+}
+
+.ctaWrapper {
+	composes: g-focus-ring-from-box-shadow from global;
+	align-items: center;
+	display: flex;
+	gap: 6px;
+	color: var(--white) !important;
+
+	& .ctaIcon {
+		transition: 0.2s ease-in;
+	}
+}
+
+.solutionGradient {
+	background: var(--gradient);
+	-webkit-background-clip: text;
```

💭 do we also want to add the non-prefixed value here?

```suggestion
	background-clip: text;
	-webkit-background-clip: text;
```